### PR TITLE
#1563 Reinitialize existing stringstream in loops instead of creating…

### PR DESCRIPTION
**Pull request by rosiereflo**
… one

### CODE REVIEW

**Comment by rosiereflo** [comment]
sonar is failing due to unrelated duplicated code

File: dynawo/sources/Solvers/Common/DYNSolverImpl.cpp [hunk 1]
```diff
@@ -165,7 +165,7 @@ Solver::Impl::printHeader() const {
 
 void
 Solver::Impl::printSolve() const {
-  std::stringstream msg;
+  stringstream msg;
```
Comment (marcochiaramello) [inline dynawo/sources/Solvers/Common/DYNSolverImpl.cpp:168]: missing 
msg.str(""); 
msg.clear();

File: dynawo/sources/Models/CPP/ModelFrequency/ModelOmegaRef/DYNModelOmegaRef.cpp [hunk 2]
```diff
@@ -549,13 +563,13 @@ ModelOmegaRef::dumpUserReadableElementList(const std::string& /*nameElement*/) c
 void
 ModelOmegaRef::setFequations() {
   for (int i = 0; i < nbMaxCC; ++i) {
-    std::stringstream f;
+    stringstream f;
```
Comment (marcochiaramello) [inline dynawo/sources/Models/CPP/ModelFrequency/ModelOmegaRef/DYNModelOmegaRef.cpp:566]: missing f.str(""), f.clear() ?

File: dynawo/sources/Models/CPP/ModelFrequency/ModelOmegaRef/DYNModelOmegaRef.cpp [hunk 3]
```diff
@@ -549,13 +563,13 @@ ModelOmegaRef::dumpUserReadableElementList(const std::string& /*nameElement*/) c
...
     f << "Synchronous area " << i << " : 0 = sum_k (omega[k] * weight[k]) - omegaRef[i] * sum_k (weight[k])";
     fEquationIndex_[i] =  f.str();
   }
 
   for (int k = 0; k < nbGen_; ++k) {
-    std::stringstream f;
+    stringstream f;
```
Comment (marcochiaramello) [inline dynawo/sources/Models/CPP/ModelFrequency/ModelOmegaRef/DYNModelOmegaRef.cpp:572]: same as above

**Review by marcochiaramello — CHANGES_REQUESTED** [review]

File: dynawo/sources/Solvers/Common/DYNSolverImpl.cpp (hunk 1 above)
Comment (rosiereflo) [reply dynawo/sources/Solvers/Common/DYNSolverImpl.cpp:168]: not required for this one as it is not in a loop

Comment (marcochiaramello) [reply dynawo/sources/Solvers/Common/DYNSolverImpl.cpp:168]: ok!

File: dynawo/sources/Models/CPP/ModelFrequency/ModelOmegaRef/DYNModelOmegaRef.cpp (hunk 2 above)
Comment (gautierbureau) [reply dynawo/sources/Models/CPP/ModelFrequency/ModelOmegaRef/DYNModelOmegaRef.cpp:566]: and not out of loop ?

Comment (rosiereflo) [reply dynawo/sources/Models/CPP/ModelFrequency/ModelOmegaRef/DYNModelOmegaRef.cpp]: done

File: dynawo/sources/Models/CPP/ModelFrequency/ModelOmegaRef/DYNModelOmegaRef.cpp (hunk 3 above)
Comment (rosiereflo) [reply dynawo/sources/Models/CPP/ModelFrequency/ModelOmegaRef/DYNModelOmegaRef.cpp]: done

**Review by marcochiaramello — APPROVED** [review]

**Pull request merged**